pkg/satellite/database.go

```diff
@@ -22,9 +22,14 @@ type DB interface {
 	Close() error
 
 	// BeginTransaction is a method for opening transaction
-	BeginTransaction(ctx context.Context) error
+	BeginTransaction(ctx context.Context) (DBTx, error)
```

Comment:
Use `BeginTx`, that way the name matches https://golang.org/pkg/database/sql/#DB.BeginTx